tortoise/tortoise.go

```diff
@@ -24,13 +24,18 @@ var (
 	errstrTooManyExceptions = "too many exceptions to base ballot vote"
 )
 
+type blockValidityUpdater interface {
```

Comment:
does it need to use updater? i thought that we can either:
- drop interface and use db directly in tortoise
- drop writes in tortoise (more changes in tests i guess)

Reply:
> does it need to use updater? 

i keep it mostly to keep the current tests unchanged.

> i thought that we can either:
> 
> * drop interface and use db directly in tortoise
> * drop writes in tortoise (more changes in tests i guess)

i can write the validity updates to db after `Updates()` is queried. that will allow dropping the interface and not changing the tests. in mesh.go i currently write to db in a defer function anyway.

Reply:
@dshulyak 
i ended up dropping writes in tortoise and changed the tests. please review. thank you